modules/monitoring-configuring-the-monitoring-stack.adoc

```diff
@@ -67,6 +67,11 @@ data:
 ----
 <1> Defines the Prometheus component and the subsequent lines define its configuration.
 
+[NOTE]
```

Comment:
we have 3 stories under the epic, they are for prometheus-operator/prometheus-adapter/kube-rbac-proxy in kube-state-metrics, all these instances should honor the global tlsSecurityProfile defined in the global apiserver resource
`# oc get APIServer cluster -oyaml`,  I am not sure if the `NOTE` section is proper, need @jan--f help to confirm

Reply:
I agree, the intention is to set `tlsSecurityProfile` for all components that use TLS, not just Prometheus. @fpetkovski can you please leave a note how far along we are with this?

@bburt-rh any thoughts on a more generic location for this note?

Reply:
In 4.10, the TLS security profile will be respected by three components in CMO: prometheus-operator, prometheus-adapter and kube-rbac-proxy. If we add additional components that serve HTTPS traffic, they would also need to reconfigure the TLS settings dynamically based on the centrally managed TLS profile. So this feature is part of the entire monitoring stack, not just Prometheus. At the moment, Prometheus itself is not part of the list of components. It only serves plain HTTP traffic and kube-rbac-proxy is responsible for adding TLS.

So far this functionality has been implemented for prometheus-operator and prometheus-adapter, and kube-rbac-proxy is being worked on at the moment.

With regards to the docs, I agree that it would be best to document this feature in a more generic way. We should convey to the end user that all components of the monitoring stack will respect the settings that are centrally configured by a cluster administrator.

Reply:
Thanks for the feedback, everyone. I will move the note to a more generic location to indicate that the entire monitoring stack will respect the TLS settings.